Mini_Project_1_Twitter.ipynb

```diff
@@ -0,0 +1,204 @@
+      "@JetBlue I thought so, I just didn't want to get stuck at Montego Bay airport. Are you not concerned about hurricane Matthew in Jamaica? (0.0, 0.0) neutral\n",
+      "@JetBlue thanks for answering (0.2, 0.2) positive\n",
+      "@JetBlue Great choice of music in Terminal 5 JFK, esp @coldplay (0.8, 0.75) positive\n",
+      "@JetBlue can you tell me when flight number 560 scheduled to depart from Kingston JA will be reschedule? (0.0, 0.0) neutral\n",
+      "@JetBlue Airways Corporation (NASDAQ:JBLU) – Broker Update - NewsDen : https://t.co/LYITm16FOE (0.0, 0.0) neutral\n",
+      "@JetBlue Flight number B6  1722 thank you! (0.0, 0.0) neutral\n",
+      "@JetBlue I want to thank you for your quick response and help on a delayed flight on Saturday. We appreciate the service! (1 of 2) (0.41666666666666663, 0.5) positive\n",
+      "@JetBlue Could you please confirm if my flight from Fort Lauderdale to Montego Bay Jamaica tomorrow hasn't been canceled? Fly 327 #Hurricane (0.8, 0.9) positive\n",
+      "@JetBlue Can you please confirm my flight from Montego Bay Jamaica tomorrow at 3 pm has not been canceled? Seems odd given the hurricane. (-0.16666666666666666, 0.25) negative\n",
+      "RT @CaseyNeistat: hey @JetBlue $12.99 for a to go wrap feels like gauging.  at JFK.  just wanted to share https://t.co/PHVIZZ0cTh (0.0, 0.0) neutral\n",
+      "Customer service @Jetblue @MCO A+ today (0.0, 0.0) neutral\n",
+      "the number of tweets that have a positive sentiment is 41 or 41.0 %\n",
+      "the number of tweets that have a neutral sentiment is 46 or 46.0 %\n",
+      "the number of tweets that have a negative sentiment is 13 or 13.0 %\n"
+     ]
+    }
+   ],
+   "source": [
+    "#$ sudo pip install pattern \n",
+    "#$ sudo pip install matplotlib\n",
+    "\n",
+    "from pattern.web import Twitter\n",
+    "from pattern.en import sentiment\n",
+    "from matplotlib import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "def sent_analysis(company, n):\n",
+    "    count_total = 0.0\n",
+    "    count_positive = 0\n",
+    "    count_neutral = 0\n",
+    "    count_negative = 0\n",
+    "    for tweet in Twitter().search(company, count = n):\n",
+    "        count_total = count_total + 1\n",
+    "        sent = sentiment(tweet.text)\n",
+    "        print tweet.text, sent,\n",
+    "        if sent[0] > 0.1:\n",
+    "            print 'positive'\n",
+    "            count_positive = count_positive + 1\n",
+    "        elif -0.1 < sent[0] < 0.1:\n",
+    "            print 'neutral'\n",
+    "            count_neutral = count_neutral + 1\n",
+    "        else:\n",
+    "            print 'negative'\n",
+    "            count_negative = count_negative + 1\n",
+    "        percent_positive = (count_positive/count_total)*100\n",
+    "        percent_neutral = (count_neutral/count_total)*100\n",
+    "        percent_negative = (count_negative/count_total)*100\n",
```

Comment:
These three lines above can be outside of the for loop. You're overriding them each time and then doing nothing with the result until after the for loop